config/src/test/java/org/springframework/security/config/doc/NicerNode.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2002-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.config.doc;
+
+import org.w3c.dom.Node;
+
+import java.util.stream.Stream;
+
+/**
+ * @author Josh Cummings
+ */
+public interface NicerNode {
```

Comment:
I'd avoid interfaces unless we need multiple implementations or if the API is made accessible to users. For those reasons, `NicerNode` (along with the Nicer* interfaces) should be removed

Reply:
I thought you might say that. :) No problem, will do.